lib/charms/tls_certificates_interface/v4/tls_certificates.py

```diff
@@ -1181,6 +1199,9 @@ def __init__(
                 Using this parameter is discouraged,
                 having to pass around private keys manually can be a security concern.
                 Allowing the library to generate and manage the key is the more secure approach.
+            renewal_relative_time (float): The time to renew the certificate relative to its
```

Comment:
We should probably document the acceptable range here.

Reply:
Added.